tests/e2e/kubernetes/kubernetes_test.go

```diff
@@ -970,3 +970,89 @@ func TestUpgradePathNonHAModeMTLSEnabled(t *testing.T) {
 		})
 	}
 }
+
+func TestUpgradePathHAModeMTLSDisabled(t *testing.T) {
+	// Ensure a clean environment
+	uninstall() // does not wait for pod deletion
+	for _, p := range supportedUpgradePaths {
+		t.Run(fmt.Sprintf("setup v%s to v%s", p.previous.runtimeVersion, p.next.runtimeVersion), func(t *testing.T) {
+			t.Run("delete CRDs "+p.previous.runtimeVersion, deleteCRD(p.previous.customResourceDefs))
+			t.Run("delete CRDs "+p.next.runtimeVersion, deleteCRD(p.next.customResourceDefs))
```

Comment:
This is needed because of https://github.com/dapr/cli/issues/656 ? Are we really not going to address that?

Reply:
See https://github.com/dapr/cli/issues/656#issuecomment-840014032. If it is reopened and resolved then the tests can change. Till then this has to remain.